src/main/java/org/llorllale/cactoos/matchers/MatchesRegex.java

```diff
@@ -53,7 +53,8 @@ public MatchesRegex(final Text regex) {
             new TextMatcher(
                 new MatcherOf<>(
                     (String act) -> act.matches(regex.asString()),
-                    regex
+                        desc -> desc.appendText(regex.asString()),
+                        (actual, desc) -> desc.appendValue(actual)
```

Comment:
@ruca905 for all those matchers you modified, you need to add some relevant message to the desc in the two func here. For example here it's about a regex. Same for all the other ones (or if you don't want to do them right now, do just a few of them as I proposed earlier, but do them completely).